examples/default.yaml

```diff
@@ -121,6 +121,8 @@ ssh:
   # 🟢 Builtin default: true
   loadDotSSHPubKeys: null
   # Forward ssh agent into the instance.
+  # The ssh agent socket can be mounted in a container at the path `/run/host-services/ssh-auth.sock`.
```

Comment:
Permission should be explained too

Reply:
What should I mention about the permission?

Reply:
"The socket is accessible by the non-root user" might be ok

Reply:
I'm not sure that matters in the container.

```
╰─ lima nerdctl run -it -v /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock -e SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock golang:latest bash
root@7730a73568de:/go#
root@7730a73568de:/go# ls -l /run/host-services/ssh-auth.sock
srwxrwxr-x 1 root root 0 Feb  2 02:31 /run/host-services/ssh-auth.sock
```

Reply:
Oh, I guess you mean from inside the VM.

Reply:
Done.